### DISCUSSION TITLE
Content rating updater job is not running

### DISCUSSION THREAD

**Discussion by jayrajk112** [Sunbird-Obsrv/Community#140] [Issues]
Hi Team,

There is one job on lower spark content-rating-updater is currently not working. We are trying to run manually but we didn't get any log for this job.
can you please help us to know the complete workflow process of content rating job?
please let us know where can we get the code for the same. please share the link of githum code for rating job?
can u let us know there any customize script for the same?
 
Thanks,
Jayraj Kakde

**Comment by jayrajk112** [comment]
Please any update on this

**Reply by SanthoshVasabhaktula** [reply]
@sowmya-dixit - Please respond on this

**Reply by sowmya-dixit** [reply]
@jayrajk112 
Here is the Content rating job workflow - 
    We have feedback telemetry events ingested into Druid daily and this job queries for newly rated contents, computes the latest average rating on those contents and update contents metadata with computed rating using system update API.

Logs for this will be available under `/mount/data/analytics/logs/data-products/<execution-date>-job-execution.log`

Github link for code - https://github.com/Sunbird-Obsrv/sunbird-core-dataproducts/blob/release-5.1.3/batch-models/src/main/scala/org/ekstep/analytics/updater/UpdateContentRating.scala

Please re-run the job, check for logs under above mentioned path and let us know if you face any issues.

**Reply by jayrajk112** [reply]
Thanks for your response

**Reply by kamal-singhal** [reply]
@sowmya-dixit @jayrajk112   Thank you very much to provide response. we have seen the system update API has update the data in database. 
lp.system.update.base.url="http://localhost:8080/learning-service/system/v3/content/update"

Can you let us know above API update the data only Neo 4j or any other data base also. we have checked data in database about content rating, only get the update data in Neo4j and elastic search not in another data base like Cassandra.

Please help us for the same.

**Reply by sowmya-dixit** [reply]
Yes - System Update API updates data only in Neo4j and elastic serach.

**Reply by kamal-singhal** [reply]
@sowmya-dixit Thanks for the response.

If we need to update content rating data from neo4j to Cassandra. How can this be possible. How has data been updated in Cassandra before? Is there any specific job which syncs data from neo4j to Cassandra.

**Reply by sowmya-dixit** [reply]
@kamal-singhal I hope more details on neo4j to Cassandra updates are available here in this discussion - https://github.com/orgs/Sunbird-Knowlg/discussions/173
@jayrajk112  If content rating updater job issue is resolved, can we close this?
